raisinbread/README.md

```diff
@@ -68,8 +68,8 @@ directly on the mysql command line.
 
 ```bash
 cat SQL/0000-00-00-schema.sql \
-    SQL/0000-00-01-Modules.sql \
-    SQL/0000-00-02-Permission.sql \
+    SQL/0000-00-01-Permission.sql \
+    SQL/0000-00-02-Modules.sql \
```

Comment:
@lbewsell as you can see in the SQL directory (https://github.com/aces/Loris/tree/main/SQL) the correct files are 
```
SQL/0000-00-01-Modules.sql \
SQL/0000-00-02-Permission.sql \
```

This is actually a change that took place in the latest version of LORIS which should be released soon. This documentation points to the latest updated information.

The changes happened here #7327

Reply:
Thanks for explaining that! I'm going to go ahead close this PR